commands/global/new.js

```diff
@@ -118,7 +118,7 @@ module.exports.handler = function handler(options) {
       require('codemod-cli').runTransform(
         __dirname,
         process.argv[2]       /* transform name */,
-        process.argv.slice(3) /* paths or globs */
+        ...process.argv.slice(3) /* paths or globs */
```

Comment:
I actually intended to pass an array (instead of splat them out) specifically to simplify the usage on "both sides" (note how you also had to change the implementation of `runTransform` to do `...args`). If we just pass `process.argv.slice(3)` here and do `args` there you are good to go...

Reply:
Done!